browser/brave_stats/first_run_util_unittest.cc

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_stats/first_run_util.h"
+#include "base/time/time.h"
+#include "brave/components/constants/pref_names.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/testing_pref_service.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace brave_stats {
+
+class FirstRunUtilTest : public testing::Test {
+ public:
+  FirstRunUtilTest() = default;
+  ~FirstRunUtilTest() override = default;
+
+  void SetUp() override {
+    local_state_.registry()->RegisterTimePref(kReferralAndroidFirstRunTimestamp,
```

Comment:
call `RegisterPrefsForBraveReferralsService` here instead to ensure we're using the same default value for testing